messagequeues/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Text;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Topshelf;
+
+namespace messagequeues;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+
+        var exitCode = HostFactory.Run(x => 
+        {
+            x.Service<ImageTransformation>(s => {
+                s.ConstructUsing(imageTransformationService => new ImageTransformation());
+                s.WhenStarted(imageTransformationService => imageTransformationService.Start());
+                s.WhenStopped(imageTransformationService => imageTransformationService.Stop());
+            });
+
+            x.RunAsLocalSystem();
+            x.SetServiceName("imageTransformationService");
+            x.SetDisplayName("image Transformation Service");
+            x.SetDescription("DataCaptureService");
+        });
+
+        int exitCodeValue = (int)Convert.ChangeType(exitCode, exitCode.GetTypeCode());
+        Environment.ExitCode = exitCodeValue;
+
+        // For other Service
+
+        // var exitCode = HostFactory.Run(x => 

```

Comment:
let's move this to another project, or just use command line parameter that will decide what service to run, the task requires execution of two services at the same time